systemtests/src/main/java/io/enmasse/systemtest/utils/AddressSpaceUtils.java

```diff
@@ -219,6 +220,30 @@ public static void waitForAddressSpaceDeleted(AddressSpace addressSpace) throws
         }
     }
 
+    public static Endpoint getServiceEndpointByName(AddressSpace addressSpace, String endpoint, String protocol) {
```

Comment:
@lulf I have same method in my smoke messaging branch on cluster so I don't think we need 2 same methods